nflow-engine/src/main/java/io/nflow/engine/internal/executor/WorkflowStateProcessor.java

```diff
@@ -178,7 +177,7 @@ private void rescheduleUnknownWorkflowType(WorkflowInstance instance) {
     logger.warn("Workflow type {} not configured to this nFlow instance - rescheduling workflow instance", instance.type);
     instance = new WorkflowInstance.Builder(instance).setNextActivation(now().plusMinutes(unknownWorkflowTypeRetryDelay))
         .setStatus(inProgress).setStateText("Unsupported workflow type").build();
-    workflowInstanceDao.updateWorkflowInstance(instance);
+    workflowInstanceDao.updateWorkflowInstance(instance, null);
```

Comment:
The `null` will clear the current start time from the db.

Reply:
If started is set in db, it is never updated by instance update query